### PR TITLE
MTDSIM-44

### PR DESCRIPTION
**Pull request by Vobrymn**
Update to /simulate endpoint doc to reflect the new return object and its details

### CODE REVIEW

**Comment by benvarian** [comment]
is this ready to be looked at again

File: server/Documentation/docs/backend/endpoints.md [hunk 1]
```diff
@@ -82,100 +91,55 @@ This endpoint runs the `simulate_without_saving` function that can be found in `
...
-          "totalServices": 7,
-          "totalUsers": 5
-        },
-        "id": 1,
-        "layer": 0,
-        "subnet": 0
-      },
-      ...
-    ]
-  }
```
Comment (nicodeas) [inline server/Documentation/docs/backend/endpoints.md:175]: My stuff is still deleted.

File: server/Documentation/docs/backend/endpoints.md [hunk 2]
```diff
@@ -82,100 +91,55 @@ This endpoint runs the `simulate_without_saving` function that can be found in `
...
-```
+- MTD Record
+```json
 {
-  "finishTime":int (optional),
-  "mtdInterval":int,
-  "scheme":string,
-  "totalNodes":int,
-  "seed": int (optional)
+'name': {0: 'CompleteTopologyShuffle', 1: 'OSDiversity', 2: 'OSDiversity'},
```
Comment (nicodeas) [inline server/Documentation/docs/backend/endpoints.md:97]: heres a field called name?

Comment (nicodeas) [reply server/Documentation/docs/backend/endpoints.md:97]: could you leave it in the format that it was left before? i think this is complicated

File: server/Documentation/docs/backend/endpoints.md [hunk 3]
```diff
@@ -82,100 +91,55 @@ This endpoint runs the `simulate_without_saving` function that can be found in `
...
+'start_time': {0: 0.0, 1: 200.3685748174762, 2: 400.9527232348661},
+'finish_time': {0: 121.81486938465021, 1: 280.6090999799783, 2: 481.0931564591807},
+'duration': {0: 121.81486938465021, 1: 80.24052516250208, 2: 80.14043322431462},
+'executed_at': {0: 'network', 1: 'application', 2: 'application'}
```
Comment (nicodeas) [inline server/Documentation/docs/backend/endpoints.md:101]: is there a field for this?

File: server/api/app.py
Comment (nicodeas) [inline server/api/app.py:1]: 👍🏻

**Review by nicodeas — CHANGES_REQUESTED** [review]

**Comment by benvarian** [comment]
is this issue ready to be reviewed again, if so please close all of nicks requirements and itll be looked at soon

File: server/Documentation/docs/backend/endpoints.md (hunk 2 above)
Comment (Vobrymn) [reply server/Documentation/docs/backend/endpoints.md:97]: This is the simplest i can make it if you want a complete sample response for each of the things we are sending to the front end. before you only had it for the network graph.

File: server/Documentation/docs/backend/endpoints.md (hunk 3 above)
Comment (Vobrymn) [reply server/Documentation/docs/backend/endpoints.md:101]: a field?

File: server/Documentation/docs/backend/endpoints.md (hunk 1 above)
Comment (Vobrymn) [reply server/Documentation/docs/backend/endpoints.md:175]: everything in main is here, not sure whay im missing

**Pull request merged**